### ISSUE TITLE
Unable to connect to MySql v8.0 Homebrew

### ISSUE THREAD

**Issue by shashiks** [flyway/flyway#2092]
###### Which version and edition of Flyway are you using?

**I have tried
flyway-commandline-5.1.4-macosx-x64.tar.gz
and
flyway-commandline-5.1.4.zip (without JRE)**

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)
**N/A**

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
**Command-Line**

###### Which database are you using (type & version)?
**MySQL Server version: 8.0.11 Homebrew
installed using homebrew**

###### Which operating system are you using?
**Max OS High Sierra  10.13.1 (17B1003)**

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

**Since there is no shell script to execute even in the mac version of command Line client. I ran the following command from the console**

`java  -cp "lib/*:drivers/*" org.flywaydb.commandline.Main migrate`

**The conf file has following values:**

```
flyway.url=jdbc:mysql://localhost:3306/mydb
flyway.user=mydbUser
flyway.password=mydbUser123
```
Not sure why it shows `using password (NO)` in the connection error. I am postitively sure there is a password in the conf. 
Other approaches tried:
    Tried providing the password as command prompt values. But same problem.
    Tried with root user too.

###### What did you expect to see?
**It should connect to the `mydb` which is mentioned in the configs using the id and password.**

###### What did you see instead?
```
ERROR:
Unable to obtain connection from database (jdbc:mysql://localhost:3306/mydb) for user 'mydbUser': Access denied for user 'mydbUser'@'localhost' (using password: NO)
---------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : 28000
Error Code : 1045
Message    : Access denied for user 'mydbUser'@'localhost' (using password: NO)
```

**Comment by shashiks** [comment]
When I enabled debugging with `-X` option I could see that it was using MariaDB connector for a mysql url and instead of complaining maybe for wrong class or something it errored out as failure to connect. 
I resolved it by putting in a mysql driver which solved my problem.

I will leave this open if you think it makes sense to ensure the code looks for a mysql default driver and fails on not having it, instead of using mariadb as default, unless specified.
